x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.tsx

```diff
@@ -245,6 +246,7 @@ interface CloudPosturePageProps<TData, TError> {
 export const CloudPosturePage = <TData, TError>({
   children,
   query,
+  hasFindings = true,
```

Comment:
why would we set `hasFindings` to true by default? `Configurations`, and `Vulnerabilities`  also uses the `CloudPosturePage` component. Will this change impact the findings configurations page?

Reply:
We set it to true for now as its a new parameter and setting it to true will basically allows the code runs like before (skipping the new if statement using hasFindings)